elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -174,14 +165,22 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
       whenReady(resFut) { res =>
         res.jsonStr should include("doc3")
         res.jsonStr should include("doc4")
-        res.jsonStr should not include("doc5")
+        res.jsonStr should not include ("doc5")
```

Comment:
I think the tradition is not to leave a space between the matcher keyword and the value. Either way, we should make the code style consistent.